check_diff/src/lib.rs

```diff
@@ -270,3 +362,41 @@ pub fn compile_rustfmt(
         feature_runner,
     });
 }
+
+pub fn search_for_rs_files(repo: &Path) -> impl Iterator<Item = PathBuf> {
```

Comment:
Would be nice to add a doc comment for `search_for_rs_files`, especially because it's marked as `pub`.